tests/bpftrace.cpp

```diff
@@ -1258,40 +1248,34 @@ TEST_F(bpftrace_bad_btf, parse_invalid_btf)
   EXPECT_FALSE(bpftrace.has_btf_data());
 }
 
-TEST(bpftrace, add_probes_rawtracepoint)
+TEST_F(bpftrace_btf, add_probes_rawtracepoint)
 {
-  StrictMock<MockBPFtrace> bpftrace;
-  parse_probe("rawtracepoint:sched_switch {}", bpftrace);
+  auto bpftrace = get_strict_mock_bpftrace();
+  parse_probe("rawtracepoint:event_rt {}", *bpftrace);
 
-  ASSERT_EQ(1U, bpftrace.get_probes().size());
-  ASSERT_EQ(0U, bpftrace.get_special_probes().size());
+  ASSERT_EQ(1U, bpftrace->get_probes().size());
+  ASSERT_EQ(0U, bpftrace->get_special_probes().size());
 
-  std::string probe_orig_name = "rawtracepoint:sched_switch";
-  check_rawtracepoint(bpftrace.get_probes().at(0),
-                      "sched_switch",
-                      probe_orig_name);
+  std::string probe_orig_name = "rawtracepoint:*:event_rt";
+  auto probe = bpftrace->get_probes().at(0);
+  EXPECT_EQ(ProbeType::rawtracepoint, probe.type);
+  EXPECT_EQ("event_rt", probe.attach_point);
+  EXPECT_EQ("rawtracepoint:*:event_rt", probe.orig_name);
+  EXPECT_EQ("rawtracepoint:event_rt", probe.name);
```

Comment:
Why is this not `rawtracepoint:vmlinux:event_rt`?

Reply:
because this function `event_rt` is coming from our data_source.c file that we load as the test BTF.